src/test/java/org/carlspring/ioc/mock/PropertyHolderWithIntAndLongProperties.java

```diff
@@ -0,0 +1,144 @@
+package org.carlspring.ioc.mock;
+
+import org.carlspring.ioc.PropertiesResources;
+import org.carlspring.ioc.PropertyValue;
+
+/**
+ * @author basdewitte
+ */
+
+@PropertiesResources(resources = { "META-INF/properties/typed.properties" })
+public class PropertyHolderWithIntAndLongProperties
+{
+
+    @PropertyValue(key = "prim.int")
+    protected int primInt;
+
+    @PropertyValue(key = "prim.long")
+    protected long primLong;
+
+    @PropertyValue(key = "prim.double")
+    protected double primDouble;
+
+    @PropertyValue(key = "prim.float")
+    protected float primFloat;
+
+    @PropertyValue(key = "prim.bool")
+    protected boolean primBool;
```

Comment:
This is just a code style remark really (it had nothing to do with the quality of your work) -- I normally use camel hump variables with full names... So this would be `primitiveBoolean`. This makes sure that there no confusions. Alternatively, in methods, if it's very clear what the name is, I do things like `InputStream is = ...`.